### DISCUSSION TITLE
Not upload to google cloud storage at version ^2.0

### DISCUSSION THREAD

**Discussion by dangtinh97** [spatie/laravel-google-cloud-storage#37] [Q&A]
i use version 2.0.2

'apiEndpoint' => 'https://storage.googleapis.com/,
 it is working

**Comment by jonjieviduya** [comment]
I'm currently facing this issue where there is no returned error but it is also not uploading to gcs. Where should I put that code @dangtinh97?

**Comment by devnilayshah** [comment]
@dangtinh97 I am facing the same issue. Is there any chance of stable release?

**Comment by jonjieviduya** [comment]
@devnilayshah try to add `'throw' => true` in your **filesystems.php** in config folder. It should be under `'gcs'` key
